x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -288,6 +288,9 @@ clear_scroll, search, suggest, tv).
 `read_cross_cluster`::
 Read-only access to the search action from a <<cross-cluster-configuring,remote cluster>>.
 
+`read_ingest_pipelines`::
+Read-only access to ingest pipline (get, simulate).
+
```

Comment:
This is a cluster privilege, but it is documented under index privileges.

Reply:
Fixed